pipeline/unified/channel/sign-release-package.yaml

```
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+parameters:
+    filePattern: null
+    inlineSignParams: null
+    signedArtifactName: null
+    platform: null
+
+steps:
+    - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+      displayName: 'Send to ESRP Code Signing service'
```

Should we make this a bit more specific? Maybe
```suggestion
      displayName: 'ESRP signing installer'
```